packages/patternfly-4/react-core/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import styles from '@patternfly/patternfly/components/Breadcrumb/breadcrumb.css';
+import { css } from '@patternfly/react-styles';
+
+export interface BreadcrumbProps extends React.HTMLProps<HTMLElement> {
+  /** Children nodes be rendered to the BreadCrumb. Should be of type BreadCrumbItem. */
+  children?: React.ReactNode;
+  /** Additional classes added to the breadcrumb nav. */
+  className?: string;
+  /** Aria label added to the breadcrumb nav. */
+  'aria-label'?: string;
+}
+
+export const Breadcrumb: React.FunctionComponent<BreadcrumbProps> = ({
+  children = null,
+  className = '',
+  'aria-label': ariaLabel = '',
```

Comment:
Is there a reason this does not default to Breadcrumb like the current implementations?

Reply:
I still think this should default to Breadcrumb like it did before.